fastlane/Fastfile

```diff
@@ -1219,10 +1295,62 @@ def inject_buildkite_analytics_environment(xctestrun_path:)
   File.write(xctestrun_path, Plist::Emit.dump(xctestrun))
 end
 
-# Returns the version of the app based on the specified xcconfig file
+
```

Comment:
I created a test lane that verified these values are being calculated and formatted as expected (given a `VERSION_LONG` of `15.5.2` and a `VERSION_SHORT` of `15.5.2.4:

<img width="377" alt="Screenshot 2023-09-28 at 11 35 30 AM" src="https://github.com/woocommerce/woocommerce-ios/assets/17955542/240e827c-d1eb-472d-b801-48ab4381efd8">